modules/gsl/4.0.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+  "integrity": "sha256-65H8sQpqpcyx0iTgelbI7P/pobtgH6GEgnbsRqIgC/s=",
```

Comment:
Please add patch_strip = 0 in source.json

Reply:
done :)